Homebrew/Resolute Spellbook/spellbook.json

```diff
@@ -4650,6 +4650,34 @@
     "name": "(SW) Disintegrate",
     "level": 6,
     "school": "Tech",
+    "automation": [
+      {
+        "type": "target",
+        "target": "each",
+        "effects": [
+          {
+            "type": "save",
+            "stat": "dex",
+            "fail": [
+              {
```

Comment:
You're missing the closing bracket to this

Reply:
fixed